components/script/dom/webidls/Headers.webidl

```diff
@@ -4,7 +4,7 @@
 
 // https://fetch.spec.whatwg.org/#headers-class
 
-typedef (Headers or sequence<sequence<ByteString>> or MozMap<ByteString>) HeadersInit;
+typedef (Headers or sequence<sequence<ByteString>> or record<DOMString, ByteString>) HeadersInit;
```

Comment:
The change from MozMap to record makes me uncomfortable; we definitely have not done any of the work from https://bugzilla.mozilla.org/show_bug.cgi?id=1330699 .